addons/nuxeo-platform-pdf-utils/src/test/java/org/nuxeo/ecm/platform/pdf/tests/PDFTextExtractorTest.java

```diff
@@ -29,24 +40,20 @@
 import org.nuxeo.ecm.automation.OperationChain;
 import org.nuxeo.ecm.automation.OperationContext;
 import org.nuxeo.ecm.automation.test.AutomationFeature;
+import org.nuxeo.ecm.core.api.CoreSession;
 import org.nuxeo.ecm.core.api.DocumentModel;
 import org.nuxeo.ecm.core.api.impl.blob.FileBlob;
-import org.nuxeo.ecm.core.api.CoreSession;
 import org.nuxeo.ecm.platform.pdf.PDFTextExtractor;
 import org.nuxeo.ecm.platform.pdf.operations.PDFExtractTextOperation;
+import org.nuxeo.runtime.test.runner.ConditionalIgnoreRule;
 import org.nuxeo.runtime.test.runner.Deploy;
 import org.nuxeo.runtime.test.runner.Features;
 import org.nuxeo.runtime.test.runner.FeaturesRunner;
-import javax.inject.Inject;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
```

Comment:
We're trying to not change too much code for backports, format/cleanup changes shouldn't be part of a backported commit. Here it's not a real issue as it's a test class, but it should be avoided.